### ISSUE TITLE
How to upload GraphQL APIs with support for subscription to Google App Engine and Compute Engine

### ISSUE THREAD

**Issue by softpreneur** [benawad/production-web-app-checklist#1]
Great work! Am a big fan, can you help out with a step by step guide on how to setup GAE and GCE to work with GraphQL subscriptions... I have search everywhere on the internet for some weeks now but i can't find solution. Am using Node.js + Express.js + graphql-subscriptions + subscriptions-transport-ws + apollo-server-express. Thanks in advance

**Comment by benawad** [comment]
Do you know how to get subscriptions working locally?

**Comment by softpreneur** [comment]
Yes I do, my subscriptions are currently working on localhost perfectly. But everything ease one I upload on Google Compute Engine. Same for Google App Engine

**Comment by benawad** [comment]
Do your mutations and queries work on Google app engine?

**Comment by softpreneur** [comment]
Mutations and queries work fine on both

**Comment by benawad** [comment]
What error do you get with the subscriptions?

**Comment by softpreneur** [comment]
No errors, this is the project on GitHub, github.com/motionboy/sumary

**Comment by benawad** [comment]
So no errors happen, but the subscription doesn't fire? Have you tried
subscribing with GraphiQL too, to check if something is messed up on your
frontend?

**Comment by softpreneur** [comment]
Yes, I have tried it in localhost everything work fine, after creating compute engine VM and deploy app to it on GCP, the subscriptions will stop. Do u have any tutorials that act as a guide for me to deploy my GraphQL API for production and make subscriptions work? Pls any helpful video or resource that covers deploying GraphQL with subscriptions enabled will be appreciated

**Comment by benawad** [comment]
Sorry, I don't have anything. I haven't tried subscriptions in production
but I would assume they work. I would try GraphiQL while your server is on
Google cloud compute

**Comment by softpreneur** [comment]
Okay. Thanks a lot for your responses..... it means a lot to me. Keep up the good work, I send endpoint to you once it put on live server later ... thanks once more
